src/components/Label/Label.js

```diff
@@ -19,6 +22,19 @@ export default class Label extends React.Component {
       }
     }
 
+    const info = this.props.info
+      ? (
+        <div className="chi-label__help"
```

Comment:
Let's add this className to classes.js as well, please

Reply:
class name is already in STAT_CLASSES reused here